### ISSUE TITLE
player: Live TV + display mode switching causes long delay when using skip command

### ISSUE THREAD

**Issue by chrislawrence** [plexinc/plex-media-player#754]
### Test environment
**PMS Version: 3.53.7**
**PMP Version: 2.12.1**
**Platform (Windows/OSX/Embedded): Windows 10**

### Steps to reproduce
1. Set ‘Allow display mode switching’ to true
2. Watch live TV stream
3. Use skip back/forward buttons

### Current behaviour
1. If the refresh rate or resolution of the live stream are different to that of the default GUI then each time the skip button is pressed it triggers reset of the refresh rate causing the screen to be blank for 2-3 seconds. This causes a much longer pause when skipping than if display mode switching preference is off.

### Expected behavior
1. Refresh rate/resolution remains constant when skipping as occurs with playback of any other source.

**Comment by dan-the-man-nz** [comment]
i can confirm this is happening with live tv

for example, gui us 60Hz
play live tv 50Hz
skip back
PMP switches to 60Hz for a moment, then back to 50Hz and resumes playback

it is not doing this with recorded tv or other videos

here is a log showing this happening

[PlexMediaPlayer.log](https://github.com/plexinc/plex-media-player/files/2113524/PlexMediaPlayer.log)

**Comment by mseeley** [comment]
Hi folks. Sorry for the late reply.

Does this also occur when seeking during _transcoded_ playback of typical movies and TV shows on PMS? IIRC seeking in live TV forces a new transcode session. Perhaps this is being interpreted in PMP as "stop and return to the default refresh rate" before it the next session kicks off.

**Comment by dan-the-man-nz** [comment]
you are right, this does occur as you said,  the same thing happens if you force PMP to transcode a typical tv show .mkv

that is not ideal at all.

**Comment by mseeley** [comment]
Thanks @dan-the-man-nz. 👍 

Compounding bugs here. Troubles in the PMP host/MPV cause us to avoid direct playing. That in turn tickled an existing refresh rate bug.

We'll need help getting the refresh rate bug/behavior worked around.
